CHANGELOG.md

```diff
@@ -4,6 +4,9 @@
 
 ### Breaking changes
 
+* Removes testing for Python 3.7.
```

Comment:
I don't think this is a user-facing change - rather, the package doesn't support 3.7 anymore. That's worth mentioning here.